less/index.less

```diff
@@ -54,6 +54,192 @@ table {
 }
 
 html, body {
-    height: 100%;
+	max-width: 880px;
+	height: auto;
+	width: 100%;
+	margin: 0 auto;
+	background: rgb(94, 54, 2);
+	
 }
 
+//variables
+
+@stem: forestgreen;
+@skin: red;
+@flessh: #FFE5B4;
+@radius: 5px;
+
+
+
+.containter{
```

Comment:
It looks like you misspelled container here, so these styles are not being applied